packages/bruno-electron/src/utils/regex.js

```diff
@@ -0,0 +1,5 @@
+const unsafePathCharacters = /[<>:"/\\|?*]+/;
```

Comment:
I think the square brackets should be included as well.
like, `/[<>:"/\\|?*\[\]]+/;`
cc. https://github.com/usebruno/bruno/pull/1811/files